kornia/augmentation/augmentations.py

```diff
@@ -125,15 +147,13 @@ class RandomVerticalFlip(RandomFlip):
     """
 
     def __init__(self, p: float = 0.5, return_transform: bool = False) -> None:
-        super(RandomVerticalFlip, self).__init__()
-        self.p = p
-        self.return_transform = return_transform
+        super(RandomVerticalFlip, self).__init__(p=p, return_transform=return_transform)
 
-    def forward(self, input: UnionType) -> UnionType:  # type: ignore
-        return self.forward_flip(input, random_vflip)
+    def forward(self, input: UnionType, params: dict = None) -> UnionType:  # type: ignore
```

Comment:
using mypy we can be more specific about the expected type like: `Optional[Dict[str, torch.Tensor]] = None`